### PR TITLE
update development Mode setup

### PR DESCRIPTION
**Pull request by mehmood86**
The current instructions for Development Mode setup is not properly updated. It simply breaks the chain of steps when setting up development environment. 

Therefore, I suggest to add some additional git commands. This will ensure proper installation/setup of development environment for run chemotion ELN in a docker container.

### CODE REVIEW

File: docs/eln/dev_docker_install.mdx [hunk 1]
```diff
@@ -142,6 +142,10 @@ The development mode setup is straight forward and requires only a few steps. Th
 ```
 cd ~
 git clone https://github.com/ptrxyz/chemotion.git
+git remote update
+git fetch --all
+git pull --all
```
Comment (JanCBrammer) [inline docs/eln/dev_docker_install.mdx:147]: I think `fetch` and `pull` are redundant. `pull` does a `fetch` before merging branches.

Comment (mehmood86) [reply docs/eln/dev_docker_install.mdx:147]: git fetch tells your local git to retrieve the latest meta-data from the remote (yet doesn’t do any file transferring. It’s more like just checking to see if there are any changes available).

git pull on the other hand does that AND brings (copy) of those changes from the remote repository.

Should we then only keep "git pull --all" and ignore "git fetch --all"

Comment (JanCBrammer) [reply docs/eln/dev_docker_install.mdx:147]: > Should we then only keep "git pull --all" and ignore "git fetch --all" ?

Sounds good :+1:

**Comment by JanCBrammer** [comment]
Closing with 3ea223aa289fedc5048bda3de5ad3a7bcbb73656.

**Pull request closed without merging**